shared/src/packages/orders/libs/types/order-response-dto.type.ts

```diff
@@ -0,0 +1,22 @@
+import { type DriverEntity } from '~/packages/drivers/drivers.js';
+import { type ShiftEntity } from '~/packages/shifts/shifts.js';
+import { type TruckEntity } from '~/packages/trucks/trucks.js';
+import { type UserEntityT } from '~/packages/users/users.js';
+
+import { type OrderEntity } from './order-entity.type.js';
+
+type OrderResponseDto = Omit<OrderEntity, 'shift' | 'driver' | 'truck'> & {
+  shift: {
+    id: ShiftEntity['id'];
+    driver:
+      | (Pick<
+          UserEntityT,
+          'id' | 'firstName' | 'lastName' | 'email' | 'phone'
+        > &
+          Pick<DriverEntity, 'driverLicenseNumber'>)
```

Comment:
Let's introduce a name for this type. And use it here, and here `shared/src/packages/orders/libs/types/order-entity.type.ts`.
And also here `backend/src/packages/orders/order.entity.ts`